sdk/core/core-https/review/core-https.api.md

```diff
@@ -26,7 +26,9 @@ export const bearerTokenAuthenticationPolicyName = "bearerTokenAuthenticationPol
 // @public
 export interface BearerTokenAuthenticationPolicyOptions {
     credential: TokenCredential;
+    refreshAttemptsBufferMs?: number;
```

Comment:
Duration suffixes are supposed to be `InMs`: https://azure.github.io/azure-sdk/typescript_design.html#ts-options-suffix-durations

Reply:
After some thought I changed these to be:

- `beforeTokenExpiresInMs` for the number of milliseconds before the token expires, during which we start triggering refreshes.
- `tokenRefreshIntervalInMs` for the interval between token refreshes until the token expires. Defaults to 30 seconds.

How does that sound? I really like them.